.github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml

```diff
@@ -84,8 +85,9 @@ jobs:
           USER: github-actions
           CLOUDSDK_CONFIG: ${{ env.KUBELET_GCLOUD_CONFIG_PATH}}
         uses: ./.github/actions/gradle-command-self-hosted-action
-        with:
-          gradle-command: :runners:google-cloud-dataflow-java:validatesCrossLanguageRunnerGoUsingJava
+        # TODO(https://github.com/apache/beam/issues/32492): re-enable the suite.
+        # with:
+        #   gradle-command: :runners:google-cloud-dataflow-java:validatesCrossLanguageRunnerGoUsingJava
```

Comment:
I think we can preserve this "with: gradle-command". We only need to comment out the "schedule" trigger (as you did) and pull_request_target trigger (which prevents this test suite run on release branch verification)

Reply:
Ah, great, that's better. thanks!

Reply:
Actually, we don't allow this:

* What went wrong:
Execution failed for task ':beam-test-gha:check'.
> Check failed: Fail to get the trigger path for beam_PostCommit_XVR_GoUsingJava_Dataflow.yml. Make sure it has a pull_request_target trigger.

Reply:
I suppose I could write in /dev/null to silence it

Reply:
would that work with the release script?

Reply:
nvm, i'll just remove the trigger all tests.json

Reply:
PTAL

Reply:
Lol, the script doesn't like that either. We probably should just insert some logic in https://github.com/apache/beam/blob/eb8639b013e9d820a253970ed03ac0e7cfc6e13e/.github/build.gradle#L41 to allow disabling.

Maybe we could just have a string check that we have something like `gha-check-ignore` defined somewhere in the file contents? Or we could have a separate file like `release/trigger_all_disabled_workflows.json` and check that you're either triggering on that OR on `release/trigger_all_tests.json`?

Reply:
ah, this was a guard to make sure newly added tests are exercised in release branch verification, yeah we should have an exclusion list

Reply:
we can analyze workflow name to filter out disabled workflows.